Plugins/Public/playercntl_plugin/MiscCmds.cpp

```diff
@@ -927,4 +927,120 @@ namespace MiscCmds
 		cmds->Print(L"OK\n");
 		return;
 	}
+
+	void SetPlayerHp(uint clientId, float percentage)
+	{
+		Players[clientId].fRelativeHealth = percentage;
+		HookClient->Send_FLPACKET_SERVER_SETHULLSTATUS(clientId, percentage);
+	}
+
+	void SetPlayerFuse(uint clientId, uint fuseId)
+	{
+		IObjInspectImpl *obj = HkGetInspect(clientId);
+		if (obj)
+		{
+			HkLightFuse((IObjRW*)obj, fuseId, 0.0f, 0.0f, 0.0f);
+		}
+	}
+
+	void UnsetPlayerFuse(uint clientId, uint fuseId)
+	{
+		IObjInspectImpl *obj = HkGetInspect(clientId);
+		if (obj)
+		{
+			HkUnLightFuse((IObjRW*)obj, fuseId, 0.0f);
+		}
+	}
+
+	uint CheckAdminRightAndGetTargetClient(CCmds* cmds, const wstring& charName)
+	{
+		if (!(cmds->rights & RIGHT_EVENTMODE))
```

Comment:
I don't think this is what eventmode rights are supposed to be for

Reply:
This seems exactly like sort of stuff Event Admin would want to have available, I consulted with Haste before submitting, he was fine with it.